src/ProxyManager/Generator/Util/IdentifierSuffixer.php

```diff
@@ -23,7 +23,7 @@ abstract class IdentifierSuffixer
     public const VALID_IDENTIFIER_FORMAT = '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]+$/';
     public const DEFAULT_IDENTIFIER      = 'g';
 
-    final private function __construct()
+    private function __construct()
```

Comment:
This was `final` on purpose

Reply:
reverted https://github.com/Ocramius/ProxyManager/pull/732/commits/96d5c8d1de077a0a1e0e56b4d4871c5d395be814